plugin.js

```diff
@@ -14,7 +14,8 @@ const defaultPolicy = {
 
 const defaultAdditionalOpts = {
   enabled: true,
-  hashingMethod: 'sha256'
+  hashingMethod: 'sha256',
+  hashIfUnsafeInlinePresent: true
```

Comment:
I think it would be best to infer this setting from the policy which is explicitly passed in by the dev - if they set `unsafe-inline`, then we can skip creating hashes instead of requiring them to set a setting

Reply:
Thanks for reviewing 😄! I considered inferring the hashing behavior, but I realized that might complicate things in a different way. [The default policy already contains `unsafe-inline`](https://github.com/slackhq/csp-html-webpack-plugin/blob/master/plugin.js#L11-L12), so I'd need to save a copy of the passed-in policy in addition to the [merged policy](https://github.com/slackhq/csp-html-webpack-plugin/blob/master/plugin.js#L28). Then the code that does the hashing would get a bit more complex as well, since it'd need to compare against the user-specified policy, but operate on the merged policy. If that's fine, I can go ahead and make the change.

Reply:
I think thats ok to do - really, the default policy exists simply to provide something incase the user hasn't specified the policy themselves - we can always assume the default policy will have `unsafe-inlines`, so we just have to do a check against the user defined one like you've mentioned.

Thanks!